index.js

```diff
@@ -1,5 +1,67 @@
 'use strict';
 
+var superagent = require('superagent');
+
+global.D2LAccessTokenExpiresAt = 0;
+
+function now() {
+	return Date.now()/1000 | 0;
+}
+
+function addHeaders(req) {
+	req.set('X-Csrf-Token', localStorage['XSRF.Token']);
+	req.set('X-D2L-App-Id', 'deprecated');
```

Comment:
We used to force that this header be sent but removed it. Keeping it in the library for now because not every LMS will be lenient.